recipes/jtckdint/all/conanfile.py

```diff
@@ -0,0 +1,40 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import copy, get
+from conan.tools.layout import basic_layout
+import os
+
+
+required_conan_version = ">=2.0"
+
+
+class JtckdintConan(ConanFile):
+    name = "jtckdint"
+    description = "C23 Checked Arithmetic"
+    license = "ISC"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/jart/jtckdint"
+    topics = ("integer", "checked", "arithmetic", "header-only")
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
+    no_copy_source = True
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def package_id(self):
+        self.info.clear()
+
+    def validate(self):
+        check_min_cppstd(self, 11)
```

Comment:
This is checking the cppstd, I think it can be safely removed

Reply:
Or is this actually a C++ library? I got a bit confused on the readme

Reply:
@AbrilRBS 
Oops.
I understood it incorrectly.
I needed C11, not C++11.
Deleted.
Thanks a lot!